### PR TITLE
add db migration doc

### PR DESCRIPTION
**Pull request by jakekaplan**
Adds instructions on how to run database migrations and add a new database migration

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: docs/concepts/database.md [hunk 1]
```diff
@@ -121,3 +121,103 @@ The following database versions are required for use with Prefect:
...
+or to downgrade to a specific revision:
+<div class="terminal">
+```bash
+prefect server database downgrade -y -r d20618ce678e
+```
+</div>
+
+### Adding a new migration
+To make changes to a table, first update the SQLAlchemy model in `src/prefect/server/database/orm_models.py`. For example
+if you wanted to add a new column to the `flow_run` table, you would add a new column to the `FlowRun` model:
```
Comment (madkinsz) [inline docs/concepts/database.md:164]: I think this section should probably be in the contributing docs

Comment (jakekaplan) [reply docs/concepts/database.md:164]: can definitely shift this over there. do you think it makes sense to link to that from this page at all?

Comment (madkinsz) [reply docs/concepts/database.md:164]: Can't hurt to link :)

**Review by zangell44 — APPROVED** [review]

File: docs/contributing/adding-db-migrations.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Adding Database Migrations
+To make changes to a table, first update the SQLAlchemy model in `src/prefect/server/database/orm_models.py`. For example
```
Comment (billpalombi) [inline docs/contributing/adding-db-migrations.md:2]:
```suggestion
To make changes to a table, first update the SQLAlchemy model in `src/prefect/server/database/orm_models.py`. For example,
```

File: docs/contributing/adding-db-migrations.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Adding Database Migrations
```
Comment (madkinsz) [inline docs/contributing/adding-db-migrations.md:1]: I think you need to add this to the index

File: docs/contributing/adding-db-migrations.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+# src/prefect/server/database/orm_models.py
+
+@declarative_mixin
+class ORMFlowRun(ORMRun):
+    """SQLAlchemy model of a flow run."""
+    ...
+    new_column = Column(String, nullable=True) # <-- add this line
+```
+
+Next, you will need to generate a new migration file. You will need to generate a new migration file for each database type.
```
Comment (billpalombi) [inline docs/contributing/adding-db-migrations.md:15]:
```suggestion
Next, you will need to generate new migration files. You must generate a new migration file for each database type. 
```

File: docs/contributing/adding-db-migrations.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+<div class="terminal">
+```bash
+prefect server database revision --autogenerate -m "add_flow_run__new_column"
+```
+</div>
+
+The `--autogenerate` flag will automatically generate a migration file based on the changes to the models. This does
+not guarantee a correct migration, so you will need to manually inspect the generated migration file to make sure
+it is accurate. Be sure to make sure to remove any commands that are not related to the change you are making.
```
Comment (billpalombi) [inline docs/contributing/adding-db-migrations.md:28]:
```suggestion
it is accurate. Be sure to remove any commands that are not related to the change you are making.
```

File: docs/contributing/adding-db-migrations.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+When adding a migration for SQLite it's important to include the following `PRAGMA` statements for both upgrade and downgrade
```
Comment (billpalombi) [inline docs/contributing/adding-db-migrations.md:30]:
```suggestion
When adding a migration for SQLite, it's important to include the following `PRAGMA` statements for both upgrade and downgrade:
```

**Review by billpalombi — COMMENTED** [review]
Light copyedits, otherwise lgtm.

**Review by madkinsz — APPROVED** [review]

**Pull request merged**